blog-post.html

```diff
@@ -1,133 +0,0 @@
-    <!--Title HTML-->
-
-    <section id=title>
-      Blog Page
-    </section>
-
-    <!------------------------------------------------------------------------------------------------->
-    <!--Posts HTML-->
-
-    <post id='post1'>
-      <div class=date>Nov 24th</div>
-      <h2>Some Post</h2>
-      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Perferendis, et ducimus. Cupiditate porro corporis
-        accusantium doloremque aspernatur recusandae ducimus distinctio, quis totam. Saepe delectus dicta tempora modi,
-      </p>
-      <div class='lineBreak'></div>
-    </post>
-    <post id='post2'>
-      <div class=date>Nov 27th</div>
-      <h2>Another Post</h2>
-      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Perferendis, et ducimus. Cupiditate porro corporis
-        accusantium doloremque aspernatur recusandae ducimus distinctio, quis totam. Saepe delectus dicta tempora modi,
-      </p>
-      <div class='lineBreak'></div>
-    </post>
-    <post id='post3'>
-      <div class=date>Nov 30th</div>
-      <h2>Some Other Post</h2>
-      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Perferendis, et ducimus. Cupiditate porro corporis
-        accusantium doloremque aspernatur recusandae ducimus distinctio, quis totam. Saepe delectus dicta tempora modi,
-      </p>
-      <div class='lineBreak'></div>
-    </post>
-    <post id='post4'>
-      <div class=date>Dec 2nd</div>
-      <h2>A Post</h2>
-      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Perferendis, et ducimus. Cupiditate porro corporis
-        accusantium doloremque aspernatur recusandae ducimus distinctio, quis totam. Saepe delectus dicta tempora modi,
-      </p>
-      <div class='lineBreak'></div>
-    </post>
-    <post id='post5'>
-      <div class=date>Dec 8th</div>
-      <h2>A Different Post</h2>
-      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Perferendis, et ducimus. Cupiditate porro corporis
-        accusantium doloremque aspernatur recusandae ducimus distinctio, quis totam. Saepe delectus dicta tempora modi,
-      </p>
-      <div class='lineBreak'></div>
-    </post>
-    <post id='post6'>
-      <div class=date>Dec 12th</div>
-      <h2>Another Post</h2>
-      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Perferendis, et ducimus. Cupiditate porro corporis
-        accusantium doloremque aspernatur recusandae ducimus distinctio, quis totam. Saepe delectus dicta tempora modi,
-      </p>
-      <div class='lineBreak'></div>
-    </post>
-    <post id='post7'>
-      <div class=date>Dec 16th</div>
-      <h2>Yet Another Post</h2>
-      <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Perferendis, et ducimus. Cupiditate porro corporis
-        accusantium doloremque aspernatur recusandae ducimus distinctio, quis totam. Saepe delectus dicta tempora modi,
-      </p>
-      <div class='lineBreak'></div>

```

Comment:
As a fun fact: these types of classes, that have a single distinct function that you use to manipulate your layout, or design are called 'utility classes'